NEMO/templates/tool_control/tool_status.html

```diff
@@ -685,9 +686,26 @@ <h4 class="media-heading">Task &amp; comment history for this tool</h4>
 		}
 	}
 
+    function update_start_button()
+	{
+        let hasPreUsageQuestions =$("#has_pre_usage_questions").val();
+        if (hasPreUsageQuestions === "True"){
```

Comment:
you can use the following "trick" and remove the hidden input
```suggestion
        if ({{ tool.pre_usage_questions|yesno:"true,false" }}){
```